apps/etterlatte-grunnlag/.nais/dev.yaml

```diff
@@ -113,5 +109,4 @@ spec:
         - application: etterlatte-pdltjenester
       external:
         - host: etterlatte-unleash-api.nav.cloud.nais.io
-        - host: pensjon-persondata-q2.dev-fss-pub.nais.io
```

Comment:
Nice!

Reply:
Bør vi også tipse persondata-teamet om at dei kan fjerne opninga på si side?

Reply:
Ja, det skal jeg gjøre!

Reply:
https://nav-it.slack.com/archives/C04PR7T1Y0G/p1720442198886019